web/src/features/projects/pages/ViewOneProjectPage.tsx

```
@@ -96,7 +96,7 @@ const ViewOneProjectPage = () => {
                     {t('manageProjectUsers')}
                   </Button>
 
-                  <Button className="m-2" variant={'danger'} onClick={() => deleteProject(params.projectId!)}>
+                  <Button className="m-2" variant={'danger'} onClick={() => deleteProject(projectId!)}>
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)